packit_service/worker/handlers/forges.py

```diff
@@ -181,6 +187,22 @@ def run(self) -> TaskResults:
 
         fas_account = command_parts[1]
 
+        if original_sender_login != self.sender_login:
+            msg = (
+                "Packit verification comment command not created by the person who "
+                "installed the application."
+            )
+            logger.debug(msg)
+            self.issue.comment(msg)
+            return TaskResults(success=True, details={"msg": msg})
+
+        return self.verify(namespace=namespace, fas_account=fas_account)
```

Comment:
agreed that the use case for doing the verification for someone else sounds iffy